### ISSUE TITLE
Provide a `kaniko auth` command, like crane

### ISSUE THREAD

**Issue by nejch** [GoogleContainerTools/kaniko#1239]
Crane provides a `crane auth` command to make auth a bit more convenient in CI environments (since https://github.com/google/go-containerregistry/commit/52c40873dc05f21db65f2de4b86cc62bc53bcc1e).

The recent change in https://github.com/GoogleContainerTools/kaniko/issues/1227 that broke everyone's builds :grin: made me wonder if kaniko could use this approach as well.

**Comment by tejal29** [comment]
Agreed this will be useful. However, I would love to prioritize #1247 before this feature!
